app/common/openmeter.go

```diff
@@ -21,7 +21,10 @@ import (
 	"github.com/openmeterio/openmeter/openmeter/namespace"
 	"github.com/openmeterio/openmeter/openmeter/sink/flushhandler"
 	"github.com/openmeterio/openmeter/openmeter/sink/flushhandler/ingestnotification"
-	"github.com/openmeterio/openmeter/openmeter/streaming/clickhouse_connector"
+	"github.com/openmeterio/openmeter/openmeter/streaming"
+	"github.com/openmeterio/openmeter/openmeter/streaming/clickhouse_connector_mv"
+	"github.com/openmeterio/openmeter/openmeter/streaming/clickhouse_connector_parse"
+	"github.com/openmeterio/openmeter/openmeter/streaming/clickhouse_connector_raw"
```

Comment:
While you are at it:

- calling the package `clickhouse`
- and hosting all three implementations in the same package

Would be much nicer.